### ISSUE TITLE
Text fields on the chat side are not opening when using a screen reader on mobile (PC is fine)

### ISSUE THREAD

**Issue by pong3147** [kwaroran/RisuAI#230]
I use voiceover on my iPhone. Since the recent 1.44.0 update, I am unable to open the chat side textfields when using a screen reader. When a textfield is active, a keyboard appears at the bottom of the screen so that I can type something, but neither the basic chat input nor the input that comes up when using auto-translation opens with VoiceOver.
The prompt and all other textfields do open, so I'm guessing that there's some kind of cursor error in the chat side of the interface. If this is possible, please fix it.
아이폰에서 보이스오버를 사용하였습니다. 최근 1.44.0 업데이트 이후로 스크린리더를 사용할 때 채팅쪽 텍스트필드를 열 수 없습니다. 텍스트필드가 활성화되어야 화면 하단에 키보드가 나타나서 원하는 내용을 입력할 수 있는데, 기본적인 채팅 입력창과, 자동번역을 사용하면 나오는 입력창 둘 다 보이스오버로 열리지 않습니다.
프롬프트 입력창이나 그 외 다른 모든 텍스트필드는 열리는 것으로 보았을 때, 채팅쪽 인터페이스에 무언가 커서 오류가 있지 않을까 짐작해봅니다. 가능하다면 수정 부탁드립니다.
